sites/main-site/src/content/hackathon-projects/hackathon-march-2025/spatial_metabolomics.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Bringing spatial metabolomics to nf-core
+category: pipelines
+intro_video: ""
+image: "/assets/images/events/2025/hackathon-march/Spat_metabo_meme.jpg"
+image_alt: "'Excited Jonah Hill' meme, where Jonah hill is screaming from excitment when he heard that spatial metabolomics workflows can be automated with nextflow"
+leaders:
+  Bisho2122:
+    name: Bishoy Wadie
+    slack: "https://nfcore.slack.com/team/U06JZ8F8LQG"
+---
+
+## Introduction
+
+**METASPACE** is the cloud computing engine for annotating metabolites, lipids, and glycans in imaging mass spectrometry data, and hosts a knowledgebase of over 13.000 public annotated datasets from over 200 labs.
+
+Most [METASPACE](https://metaspace2020.org/) users access the platform through the web app, which offers core functionalities for uploading, processing, and visualizing results. However, bioinformaticians looking to integrate METASPACE into their workflows must develop custom scripts to interact with the METASPACE Python client and submit datasets.
+
+Developing Nextflow modules will streamline this process, enabling scalable and reproducible integration of METASPACE into existing bioinformatics pipelines. This will also support METASPACE in-the-loop applications, particularly in spatial multi-omics integration workflows.
```

Comment:
```suggestion
Developing Nextflow modules will streamline this process, enabling scalable and reproducible integration of METASPACE into existing bioinformatics pipelines. 
This will also support METASPACE in-the-loop applications, particularly in spatial multi-omics integration workflows.
```
nextlfow modules or nf-core modules?